style.css

```
@@ -480,3 +474,226 @@ img {
+  height: 66px;
+  font-weight: 400;
+  line-height: 21.79px;
+  margin: 0;
+  width: inherit;
+}
+
+.signup-form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  row-gap: 40px;
+}
+
+.signup-input-container {
+  width: 100%;
+  height: 336px;
+  display: grid;
+  grid-template-columns: 250px 1fr;
+  row-gap: 20px;
+}
+
+.signup-label {
+  height: 27px;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 27.24px;
+}
+
+.signup-input {
+  width: 550px;
+  height: 42px;
+  border-radius: 8px;
+  padding: 10px;
+  border: 1px solid var(--color-form-border-red);
+  font-size: 16px;
+  line-height: 21.79px;
+  font-weight: 400;
+  background: var(--color-form-background-yellow);
+  color: var(--color-black);
+}
+
+.signup-input:focus {
+  outline: none;
+  border: 2px solid var(--color-black);
+}
+
+.signup-textarea {
+  width: 100%;
+  height: 150px;
+  border-radius: 8px;
+  padding: 10px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 21.79px;
+  background: var(--color-form-background-yellow);
+  border: 1px solid var(--color-form-border-red);
+  color: var(--color-black);
+}
+
+.signup-textarea:focus {
+  outline: none;
+  border: 2px solid var(--color-black);
+}
+
+.signup-big-button {
+  height: 60px;
+  padding: 10px 40px;
+  gap: 10px;
+  background-color: var(--color-accent-red);
+  box-shadow: 4px 4px 0 var(--color-black);
+  transition-duration: 0.3s;
+  display: flex;
+  align-items: center;
+}
+
+.signup-big-button:hover {
+  box-shadow: 12px 12px 2px var(--color-black);
+  background-color: rgba(240, 67, 165, 1);
+}
+
+.signup-big-button-label {
+  flex-grow: 1;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28.96px;
+  color: rgba(255, 254, 252, 1);
+  text-align: center;
+}
+
+@media (max-width: 800px) {
+  .signup {
+    padding: 40px 20px;
+    gap: 40px;
+    height: 670px;
+  }
+
+  .signup-container {
+    width: inherit;
+  }
+
+  .signup-section-head {
+    font-size: 24px;
+    line-height: 36.13px;
+    display: flex;
+    justify-content: center;
+  }
+
+  .PC-br {
+    display: none;
+  }
+
+  .signup-section-body {
+    font-size: 16px;
+    line-height: 19.36px;
+    height: 95px;
+  }
+
+  .signup-input-container {
+    display: block;
+    height: 300px;
+  }
+
+  .signup-label {
+    height: 19px;
+    font-size: 16px;
+    line-height: 19.36px;
+  }
+
+  .signup-input {
+    width: inherit;
+    height: 31px;
+    padding: 8px;
+    font-size: 12px;
+    line-height: 14.52px;
+    margin: 2px 0 10px 0;
+  }
+
+  .signup-textarea {
+    height: 80px;
+    padding: 8px;
+    font-size: 12px;
+    line-height: 14.52px;
+    margin: 2px 0 0;
+  }
+
+  .signup-big-button {
+    height: 40px;
+    padding: 10px 20px;
+    gap: 10px;
+  }
+
+  .signup-big-button-label {
+    height: 20px;
+    font-size: 14px;
+    line-height: 20.27px;
+  }
+
+  .signup-big-button-icon {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+/* footer */
+.footer {
+  height: 139px;
+  padding: 60px 40px;
```

```suggestion
  padding: 60px 40px;
```

ここ以外にも共通しますが、 `height` は可能な限り固定値を持たないようにしましょう。

今回は `padding` と中身のテキストの高さによって内在的に高さが決まるため、わざわざ `height` を設定する必要は無いでしょう。

`height` を設定していると困るケース：
仮に、今後Figmaが修正され、 `@2023 GrandPlayers` という文字の大きさが2倍にする必要が生じたとします。
`height` を設定していた場合は文字の大きさと `.footer` の `height` 2箇所を変更する必要がありますが、`height` を設定せず `padding` と内容によって高さが決定されていた場合、文字の大きさのみ修正すれば `.footer` 全体の高さもいい感じになります。